.storybook/recipes/CalendarCard/CalendarCard.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import { StoryObj, Meta } from '@storybook/react';
+import React from 'react';
+import { CalendarCard } from './CalendarCard';
+import { Heading, Icon, Tag, Text } from '../../../src';
+
+export default {
+  title: 'Recipes/CalendarCard',
+  component: CalendarCard,
+  decorators: [
+    (Story) => (
+      <div style={{ margin: '0.25rem' }}>
+        <Story />
+      </div>
+    ),
+  ],
+} as Meta<Args>;
+
+type Args = React.ComponentProps<typeof CalendarCard>;
+
+export const Default: StoryObj<Args> = {
+  args: {
+    children: (
+      <>
+        <CalendarCard.Body>
+          <Heading as="h3" size="body-sm">
+            <Icon
+              name="status-check-circle"
+              purpose="decorative"
+              size="1.57rem"
+            />
```

Comment:
I'm wondering if the icon and heading text should be props so we can style the spacing and keep it consistent.